addons/binding/org.openhab.binding.atlona/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,712 @@
+            <channel-group id="volume6" typeId="volumegroup6"/>
+            <channel-group id="volume7" typeId="volumegroup7"/>
+            <channel-group id="volume8" typeId="volumegroup8"/>
+            <channel-group id="volume9" typeId="volumegroup9"/>
+            <channel-group id="volume10" typeId="volumegroup10"/>
+            <channel-group id="volume11" typeId="volumegroup11"/>
+            <channel-group id="volume12" typeId="volumegroup12"/>
+        </channel-groups>
+                
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <label>IP or Host Name</label>
+                <description>IP or Host name of Atlona Pro3 1616M Switch</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="userName" type="text">
+                <label>UserName to login with</label>
+                <description>User Name to login with if Telnet Login is on</description>
+                <required>false</required>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="password" type="text">
+                <label>Password to login with</label>
+                <description>Password to login with if Telnet Login is on</description>
+                <required>false</required>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="polling" type="integer">
+                <label>Polling Interval</label>
+                <description>Interval (in seconds) to poll the actual state of the Matrix</description>
+                <default>600</default>
+                <advanced>true</advanced>
+            </parameter>            
+            <parameter name="ping" type="integer">
+                <label>Pint Interval</label>
+                <description>Ping Interval (in seconds) to keep the connection alive</description>
+                <default>30</default>
+                <advanced>true</advanced>
+            </parameter>            
+            <parameter name="retryPolling" type="integer">
+                <label>Polling Interval to try to reconnect</label>
+                <description>Interval (in seconds) to try to (re)connect to the matrix</description>
+                <default>10</default>
+                <advanced>true</advanced>
+            </parameter>            
+        </config-description>
+    </thing-type>
+
+	<channel-group-type id="primarygroup">
+	    <label>Primary Channels</label>
+	    <description>Primary Channels</description>
+	
+	    <channels>
+            <channel id="power" typeId="power"/>
+            <channel id="panellock" typeId="panellock"/>
+            <channel id="irenable" typeId="irenable"/>
+            <channel id="presetcmd" typeId="presetcmd"/>
+            <channel id="matrixcmd" typeId="matrixcmd"/>
+	    </channels>
+    </channel-group-type>  
+	  
+	  
+    <channel-group-type id="portgroup1">
```

Comment:
Without the label/description overrides - this becomes a complete mess of not knowing what port is which port (especially on the 16x16) in the paperui and I'd rather not remove it until that issue is resolved.  I'd prefer to do this because it's just alot of duplicate junk that's not needed because of the label/description issue.  What if I just update it when that issue is closed?